tests/unit/composables/PluginManager.spec.js

```diff
@@ -291,4 +313,28 @@ describe('Test composable: PluginManager', () => {
       expect(plugin.parse).toHaveBeenCalledTimes(1);
     });
   });
+
+  describe('Test function: getLanguage', () => {
+    it('should return undefined on path related to any language key', () => {
```

Comment:
`undefined` -> `null` ( since `toBeNull()` is used)

Reply:
done